pkg/bytecode/compiler.go

```diff
@@ -382,3 +433,12 @@ func (c *Compiler) compileIndexExpression(expr *parser.IndexExpression) error {
 	}
 	return nil
 }
+
+// flushBreaks will copy and return the list of break positions
+// and reset the value to an empty array.
+func (c *Compiler) flushBreaks() []int {
+	currentBreaks := make([]int, len(c.breaks))
+	copy(currentBreaks, c.breaks)
```

Comment:
You don't need to copy the breaks, as you are replacing it with a fresh slice. The new break list will not share the backing array with the old.